### ISSUE TITLE
Big number of PHP files

### ISSUE THREAD

**Issue by szepeviktor** [warfare-plugins/social-warfare#385]
Please consider class autoloading. For example you may copy it from Yoast SEO.

**Comment by szepeviktor** [comment]
OOPs, I thought the plugin is OOP.
https://github.com/warfare-plugins/social-warfare/blob/master/functions/social-networks/googlePlus.php

**Comment by Nicholas-Cardot** [comment]
Yeah, unfortunately it's mostly function based right now. However, it's on the roadmap to rebuild it all using classes and whatnot and then we can consider using autoload.

Also, unfortunately, there's not yet enough universal support for autoloading. I used autoload for a small set of classes about a year ago and as soon as I did, I got a huge wave of errors from people who were using PHP versions that don't support it.
